docker/bookkeeper/bookkeeperTeardown.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -ex
+
+HOST=`hostname -s`
+PRAVEGA_PATH=${PRAVEGA_PATH:-"pravega"}
+PRAVEGA_CLUSTER_NAME=${PRAVEGA_CLUSTER_NAME:-"pravega-cluster"}
+BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
+BK_CLUSTER_ROOT_PATH=/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}
+BK_zkServers=$(echo "${ZK_URL:-127.0.0.1:2181}" | sed -r 's/;/,/g')
+
+if [[ $HOST =~ (.*)-([0-9]+)$ ]]; then
+  NAME=${BASH_REMATCH[1]}
+  ORD=${BASH_REMATCH[2]}
+else
+  echo Failed to parse name and ordinal of Pod
+  exit 1
+fi
+
+REPLICAS=`zk-shell --run-once "get ${BK_CLUSTER_ROOT_PATH}/conf" ${BK_zkServers} | cut -d'=' -f 2`
```

Comment:
What happens if this script is executed during a network glitch? Would it be critical that the `bookieformat` command is not executed? Would that require manual intervention when trying to scale up the same Bookie to delete the cookie?

Reply:
It would be critical to execute the `bookieformat` command only when we have upgraded from the older bookkeeper version. 
In case of a fresh installation with the latest bookkeeper and bookkeeper operator, this manual invention to delete old cookies before scaling down and subsequently scaling up the bookkeeper cluster would not be required.